index.html

```diff
@@ -68,7 +68,7 @@ <h1 class="light copy">Track GeoSpatial Data Edits</h1>
 
             </div>
             <div class="col-md-offset-1 col-md-4">
-              <a href="http://download.locationtech.org/geogig/geogig-1.0-RC4.zip" class="btn download"> <span class="zipfile"></span> <div class="download-ziptxt">DOWNLOAD <br/>1.0-RC4 Release .zip</div></a>
+              <a href="https://www.locationtech.org/projects/technology.geogig/downloads/geogig-1.0.zip" class="btn download"> <span class="zipfile"></span> <div class="download-ziptxt">DOWNLOAD <br/>1.0 Release .zip</div></a>
```

Comment:
This download page has a javascript button that is backed by a load balancer (it is not a direct download the zip link). I think we are okay with that ...